R/virtualenv.R

```diff
@@ -204,7 +204,7 @@ virtualenv_default_python <- function(python) {
   if (!is.null(python))
     return(python)
 
-  config <- py_discover_config()
+  config <- py_config()
```

Comment:
This can't be `py_config()`, because that will actually cause Python to load (and the `virtualenv` family of functions shouldn't do that).